### PR TITLE
String widget respects custom border color

### PR DESCRIPTION
**Pull request by SilvanVerhoeven**

### CODE REVIEW

File: lively.ide/value-widgets.js [hunk 1]
```diff
@@ -908,10 +914,12 @@ export class StringWidget extends InputLine {
   }
 
   async onFocus (evt) {
+    if (!this.focus) this._borderColor = this.borderColor;
     super.onFocus(evt);
     if (this.readOnly) return;
     if (!this.stringTooLong) {
-      this.borderColor = Color.white.withA(0.9);
+      debugger;
```
Comment (frcroth) [inline lively.ide/value-widgets.js:921]: Remove

**Comment by merryman** [comment]
Is this one ready?

**Comment by SilvanVerhoeven** [comment]
Can now be merged @merryman 
Thanks for paying attention, @linusha :)

**Review by merryman — CHANGES_REQUESTED** [review]

File: lively.ide/value-widgets.js [hunk 2]
```diff
@@ -870,8 +870,17 @@ export class StringWidget extends InputLine {
       fill: { defaultValue: Color.transparent },
       fontColor: { defaultValue: Color.blue },
       nativeCursor: { defaultValue: 'auto' },
-      borderColor: { defaultValue: Color.transparent },
-      borderStyle: { defaultValue: 'dashed' },
+      borderColor: {
+        // will appear as white border in onFocus
+        defaultValue: Color.white.withA(0)
+      },
+      _borderColor: {
```
[the remark this answers is not in the document]
Comment (merryman) [reply lively.ide/value-widgets.js:877]: @SilvanVerhoeven Once that change is in, this is ready to go.

**Pull request merged**